packages/dapp/.env.testnet

```diff
@@ -4,3 +4,4 @@ VITE_HOTJAR_SNIPPET_VERSION=6
 VITE_LIFI_API_URL=https://staging.li.quest/v1/
 VITE_ARCX_API_KEY=bcc8c20c6f4fe15b16184a2cc8c2e1cb9125c7b574df7c0954984970a8d9de06
 VITE_CUSTOM_RPCS={ "59140": ["https://consensys-zkevm-goerli-prealpha.infura.io/v3/faf4bc4ea7344e5da5e56c55de087480"] }
+VITE_JUMPER_URL_PROD=https://jumper.exchange
```

Comment:
This value does not really change for all the stages. so putting that url in the component that uses it would make more sense IMO.